src/Type/MixedType.php

```diff
@@ -470,6 +470,11 @@ public function isArray(): TrinaryLogic
 		return TrinaryLogic::createMaybe();
 	}
 
+	public function isConstantArray(): TrinaryLogic
+	{
+		return TrinaryLogic::createMaybe();
```

Comment:
Should say no if `isArray()` says no (it's subtracted).

Reply:
pushed a commit. just like that? I did not fully understand array subtraction here

Reply:
Yeah, make sense 😊